src/test/java/htmlverification/tests/CvsPsvPrsBilingualTest.java

```diff
@@ -41,24 +37,6 @@ public void setup() throws IOException {
         String certHtmlVTP20Welsh = htmlGenerator.generate(testCertificate).get(2);
         String certHtmlVTP30Welsh = htmlGenerator.generate(testCertificate).get(3);
 
-        AtomicInteger index = new AtomicInteger(1); // Start index at 1 or 0 as needed
```

Comment:
Should this be being removed as part of this PR? how does this impact ADR Town details?